feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -358,14 +358,18 @@ fun StabilizationIcon(
  * - enabled and active: filled
  */
 @Composable
-fun LowLightBoostIcon(lowLightBoost: LowLightBoost, modifier: Modifier = Modifier) {
+fun LowLightBoostIcon(
+    lowLightBoost: LowLightBoost,
+    modifier: Modifier = Modifier,
+    active: Boolean = false
+) {
     when (lowLightBoost) {
         LowLightBoost.ENABLED -> {
             Icon(
-                imageVector = Icons.Outlined.Nightlight,
+                imageVector = if (active) Icons.Filled.Nightlight else Icons.Outlined.Nightlight,
                 contentDescription =
                 stringResource(id = R.string.quick_settings_lowlightboost_enabled),
-                modifier = modifier.alpha(0.5f)
+                modifier = modifier.alpha(if (active) 1.0f else 0.5f)
```

Comment:
Disabled states should use an alpha of `0.38f`. See https://m3.material.io/foundations/interaction/states/applying-states#4aff9c51-d20f-4580-a510-862d2e25e931